### PR TITLE
add new file assistants.py to load current assistants list

### PR DESCRIPTION
**Pull request by bzbwqz**
added new file assistants.py
modified app.py to load current assistants list
ignore the readme change

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,3 +1,33 @@
...
+10. **聊天渲染**：定义了`render_chat`函数，用于渲染聊天界面。
+
+11. **表单禁用**：定义了`disable_form`函数，用于禁用表单。
+
+12. **登录**：定义了`login`函数，用于处理登录。
+
+13. **主函数**：在主函数`main`中，首先进行了身份验证，然后创建了聊天输入框和文件上传器，接着处理用户的输入和上传的文件，最后渲染聊天界面。
+
+总的来说，这段代码的主要目的是创建一个交互式的聊天界面，用户可以通过这个界面与OpenAI的模型进行交互，并且支持文件上传和下载。此外，还支持身份验证功能。这是一个非常实用的工具，可以用于各种与OpenAI模型交互的场景。
+
```
Comment (vietnguyengit) [inline README.md:30]: maybe use English?

**Review by vietnguyengit — CHANGES_REQUESTED** [review]

File: assistants.py [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+    limit="20"
+)
+
+output = assistants.data
+
+#get all existing assistantsassistants_json = json.loads(get_extracted_data())
+assistants_list = [{"id": assistant.id, "title": assistant.name} for assistant in output] 
+
+def current_assistants():  
+    return json.dumps(assistants_list)
```
Comment (vietnguyengit) [inline assistants.py:35]: ending line, and what's the actual benefits?

**Review by vietnguyengit — CHANGES_REQUESTED** [review]

File: app.py [hunk 3]
```diff
@@ -294,7 +295,9 @@ def load_chat_screen(assistant_id, assistant_title):
 
 def main():
     # Check if multi-agent settings are defined
-    multi_agents = os.environ.get("OPENAI_ASSISTANTS", None)
+    # multi_agents = os.environ.get("OPENAI_ASSISTANTS", None)
+    # load current from assistants.py
+    multi_agents = current_assistants()
```
Comment (vietnguyengit) [inline app.py:300]: what are the actual benefits?

Comment (vietnguyengit) [reply app.py:300]: why there is a whole new file with extra 35 lines but the current `app.py` has just 1 line removed? are there uncleaned statements?

File: assistants.py (hunk 2 above)
Comment (vietnguyengit) [inline assistants.py:35]: ending line?

File: README.md (hunk 1 above)
Comment (vietnguyengit) [inline README.md:30]: is this your personal notes? Use English.

**Review by vietnguyengit — CHANGES_REQUESTED** [review]

**Comment by vietnguyengit** [comment]
just my quick look into this PR @ryo-ma , leave the careful checks for you.

**Comment by bzbwqz** [comment]
I meant to load existing assistants automatically, not to use it from .env. 
sorry for the interrupt and i will close it.

**Pull request closed without merging**